### ISSUE TITLE
Sizes Attribute Missing When Using JetPack Photon

### ISSUE THREAD

**Issue by devinsays** [ResponsiveImagesCG/wp-tevko-responsive-images#204]
I've just started testing this plugin and it looks like the sizes attribute disappears when the JetPack Photon Module is active. Since JetPack has such a large user base it's probably worth investigating to see where the conflict is happening.

Here's an example of the markup with Photon disabled:

```
<img width="720" height="269" src="http://example.com/wp-content/uploads/example-image-720x269.png" class="attachment-post-thumbnail wp-post-image" alt="Alt Text" srcset="http://example.com/wp-content/uploads/example-image-300x112.png 300w, http://example.com/wp-content/uploads/example-image-664x248.png 664w, http://example.com/wp-content/uploads/example-image-720x269.png 720w, http://example.com/wp-content/uploads/example-image-950x355.png 950w, http://example.com/wp-content/uploads/example-image.png 3588w" sizes="(max-width: 720px) 100vw, 720px" data-lazy-loaded="true" style="display: block;">
```

Here's an example of the markup with Photon enabled:

```
<img src="http://i1.wp.com/example.com/wp-content/uploads/example-image.png?fit=720%2C1200" class="attachment-post-thumbnail wp-post-image" alt="Alt Text" srcset="http://example.com/wp-content/uploads/example-image-300x112.png 300w, http://example.com/wp-content/uploads/example-image-664x248.png 664w, http://example.com/wp-content/uploads/example-image-720x269.png 720w, http://example.com/wp-content/uploads/example-image-950x355.png 950w, http://example.com/wp-content/uploads/example-image.png 3588w" data-lazy-loaded="true" style="display: block;">
```

I also posted this issue in the JetPack repo: https://github.com/Automattic/jetpack/issues/2804

**Comment by joemcgill** [comment]
Ultimately, I think Photon will need to filter the `srcset` and `sizes` attributes of images once our functionality hits core. I left a comment on the issue on the thread in their repo: https://github.com/Automattic/jetpack/issues/2804

We may also need to see if we can replace the call to `image_downsize()` when calculating `sizes` attributes to avoid this (or other odd edge cases) on our end too.

**Comment by joemcgill** [comment]
This should be resolved by 2fe9789. However, Photon should probably have a way to handle `srcset` and `sizes` internally. We'll keep tracking [the issue on their repo](https://github.com/Automattic/jetpack/issues/2804).
